apps/hash-graph/Cargo.lock

```diff
@@ -2787,7 +2822,7 @@ version = "0.4.3"
 source = "registry+https://github.com/rust-lang/crates.io-index"
 checksum = "d637b54262b1c6b8c2c88ce30fbef4e72613cb71c0e0b6fc09747052eb59a152"
 dependencies = [
- "darling",
+ "darling 0.14.4",
```

Comment:
It's a bit unfortunate, that `tsify` depends on an old version, but directly starting with the latest version is probably more important.

Reply:
Noticed that, this also means that `tsify` is pulling in `syn 1.0` (the new darling version has `syn 2.0`)

Reply:
That alone is a good reason to use the latest version 🙂